### PR TITLE
Converting music store to portable app

### PR DESCRIPTION
**Pull request by JunTaoLuo**
This needs https://github.com/aspnet/Hosting/pull/719

### CODE REVIEW

File: src/MusicStore/project.json [hunk 1]
```diff
@@ -51,14 +51,27 @@
...
       "dependencies": {
-        "NETStandard.Library": "1.5.0-*"
+        "Microsoft.NETCore.App": 
+        {
+          "version": "1.0.0-*",
+          "type": "platform"
+        }
       },
       "imports": [
         "dnxcore50",
```
Comment (pranavkm) [inline src/MusicStore/project.json]: Does it still need dnxcore50?

**Comment by pranavkm** [comment]
:shipit: once the build passes

File: test/E2ETests/project.json [hunk 2]
```diff
@@ -16,10 +16,14 @@
     "xunit": "2.1.0"
   },
   "frameworks": {
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
       "dependencies": {
+        "Microsoft.NETCore.App": 
+        {
```
Comment (davidfowl) [inline test/E2ETests/project.json]: formatting?

Comment (JunTaoLuo) [inline test/E2ETests/project.json]: Yup

File: src/MusicStore/project.json (hunk 1 above)
Comment (JunTaoLuo) [inline src/MusicStore/project.json]: Not sure. Though if we are removing dnxcore50 imports, I would imagine this isn't the only place. Feel free to file an issue in Coherence-Signed.

File: src/MusicStore/web.config [hunk 3]
```diff
@@ -4,6 +4,6 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
     </handlers>
-    <aspNetCore processPath=".\MusicStore.exe" arguments="" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="true" />
```
Comment (BrennanConroy) [inline src/MusicStore/web.config]: Don't forget to update this now :)

Comment (JunTaoLuo) [inline src/MusicStore/web.config]: Yup, will update my other PRs after https://github.com/aspnet/IISIntegration/pull/143 gets reviewed. Might also add in functional testing of standalone and portable versions a la IISIntegration.

**Comment by JunTaoLuo** [comment]
:up::date: cc @BrennanConroy 

I am duplicating all the content files required by the standalone app for now until https://github.com/dotnet/cli/pull/2493 is merged available to us. The reason for this is that we can't map the files from outside the build directory in our publish output correctly.

**Comment by BrennanConroy** [comment]
Are we tracking the cli change anywhere

**Comment by BrennanConroy** [comment]
:shipit:

**Comment by JunTaoLuo** [comment]
Tested locally and saw travis passing previously. Will create a new issue to track the removal of all the duplicate files added here once https://github.com/dotnet/cli/pull/2493 is completed.

**Pull request merged**